File: app/src/main/java/com/universityprojects/v4tch/inventorymanagement/MainActivity.java
package com.universityprojects.v4tch.inventorymanagement;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.Toast;


public class MainActivity extends AppCompatActivity {
    Button buttonLogin;
    Button buttonSign_up;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        buttonLogin = findViewById(R.id.btn_login);
        buttonSign_up = findViewById(R.id.btn_signup);

        buttonLogin.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent1 = new Intent(MainActivity.this, SignInActivity.class);
                startActivity(intent1);
            }
        });

        buttonSign_up.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent2 = new Intent(MainActivity.this, SignUpActivity.class);
                startActivity(intent2);
            }
        });
    }
}

File: app/src/main/java/com/universityprojects/v4tch/inventorymanagement/MainActivity.kt
package com.universityprojects.v4tch.inventorymanagement

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btn_login.setOnClickListener {
            val loginbutton = Intent(this@MainActivity, SignInActivity:: class.java)
            startActivity(loginbutton)
        }

        btn_signup.setOnClickListener {
            val signupbutton = Intent(this@MainActivity, SignUpActivity::class.java)
            startActivity(signupbutton)
        }



    }



}
